### DISCUSSION TITLE
Place object in timeline if ruleset is not insensitive to location(taiko)

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#20381] [Ideas] [answered]
in taiko editor. I think it is better if I can place note at timeline or not playfield

like this
![image](https://user-images.githubusercontent.com/34775378/191269997-3cf780c2-68ba-4190-9c32-75f19e29ca2c.png)

and left click to center right click to rim(need select Hit tool)

**Comment by peppy** [comment]
The timeline is made so far to be ruleset agnostic, so this wouldn't work great. We might re-think the UX at a later point but for now I don't want to change that. Is there an issue with placing objects on the main playfield instead? Is there a reason you'd prefer the timeline? Maybe we can address that reason without going in that direction.

**Reply by cdwcgt** [reply]
It would be better if the playfield could show a timeline
and do not switch time to placed object
This may be convenient to modify the state of the object in other modes, but in taiko will only be more confused when placing rhythm

**Reply by cdwcgt** [reply]
it would be nice to show where the notes would be placed on the timeline

**Comment by cdwcgt** [comment]
![image](https://user-images.githubusercontent.com/34775378/191757251-9f3db69c-2519-49d8-83f3-fe26ad6124e5.png)
```
        protected override void LoadComplete()
        {
            base.LoadComplete();
            BeginPlacement();
        }
```


I think it was an oversight that this was not included

and I made EndPlacement can be override

**final effect**

https://user-images.githubusercontent.com/34775378/191759751-01e0f578-7621-4751-88f0-f08a89c05e74.mp4

This should be a very small change

**Comment by cdwcgt** [comment, marked as answer]
https://github.com/ppy/osu/pull/20811
